extension/asapauth/go.mod

```diff
@@ -0,0 +1,47 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/extension/asapauth
+
+go 1.17
+
+require (
+	bitbucket.org/atlassian/go-asap v0.0.0-20210823164515-3d5971c40c4c
+	github.com/SermoDigital/jose v0.9.2-0.20161205224733-f6df55f235c2
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.38.0
```

Comment:
This should match the version on the `main` branch

Reply:
Fixed!